src/app/state/accountHistorySlice.ts

```diff
@@ -31,6 +31,10 @@ export interface AccountHistoryState {
   selectedTable: Tables;
   tables: Tables[];
   selectedOrdersToCancel: Record<string, Order>; // the key is `${orderRecieptAddress}_${nftRecieptId}`
+  hideOtherPairs: boolean;
+  orderHistoryAllPairs: adex.OrderReceipt[];
+  error: string | null;
+  loading: boolean;
```

Comment:
Can you explain on how you intent to use `loading`? Does it represent loading state for `fetchAccountHistory` or `fetchAccountHistoryAllPairs`, or both?

Keep in mind that we call "fetchAccountHistory" every 5 seconds to keep data updated, so if you plan on showing a loading UI using that state, we need to distinguish between calls the user has actively made and automated periodic calls.

Reply:
I thought that I need to add the cases for `rejected` and `pending` as well, should I remove these? I added `loading` solely for the `pending` case :)

Reply:
Oh sorry my bad. No you just need a fulfilled and a rejected case. I would not worry about pending yet, we can add functionality for it if loading takes long but I would remove them for now.

Reply:
I suggest to only log the error to the console in the rejected case.